core/test-runtime/client/src/lib.rs

```diff
@@ -98,6 +99,7 @@ pub struct GenesisParameters {
 	support_changes_trie: bool,
 	heap_pages_override: Option<u64>,
 	extra_storage: Vec<(Vec<u8>, Vec<u8>)>,
+	child_extra_storage: BTreeMap<Vec<u8>, Vec<(Vec<u8>, Vec<u8>)>>,
```

Comment:
Why not use a `HashMap<Vec<u8>, Vec<u8>>` directly instead of `Vec<(Vec<u8>, Vec<u8>)>`?

Reply:
I am keeping it the same way as existing code, I guess original code is this way because vec is more close to the iterator for appending, using hashmap would make thing simpler (no need to think if it is fine if keys are defined twice, it is), so switching both extra_storage and child_extra_storage could be make sense. I do not think performance really matter here (a few clone can be remove to).
So should I switch?

Reply:
I would switch, making it simpler sounds good :) And yeah, performance does not matter that much here.